src/domain/events/create/EventCreate.tsx

```diff
@@ -127,7 +128,7 @@ const EventCreate = (props: ReactAdminComponentProps) => {
         )}
         <FormDataConsumer>
           {({ formData, ...rest }) =>
-            hasInternalTicketSystem(formData) && (
+            hasInternalTicketSystem(formData) ? (
```

Comment:
trivial, but could be set to a variable instead and read from there

Reply:
Not sure how to even do that as we get `formData` just one line above 🙂 But anyways, `hasInternalTicketSystem()` is only called once, so I think this is fine.